Tasks/DownloadBuildArtifactsV0/task.loc.json

```diff
@@ -178,6 +178,15 @@
       "groupName": "advanced",
       "required": false,
       "helpMarkDown": "ms-resource:loc.input.help.parallelizationLimit"
+    },
+    {
+      "name": "checkDownloadedFiles",
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.checkDownloadedFiles",
+      "defaultValue": "$(System.ArtifactsDirectory)",
```

Comment:
I assume default value should be the same as in the task.json, seems it is required to regenerate task.loc.json

Reply:
Good catch! Fixed it.